### ISSUE TITLE
Loading cursor shakes after updating from 2.0.4 to 2.0.5

### ISSUE THREAD

**Issue by Lassebq** [ful1e5/Bibata_Cursor#158]
The cursor icon moves left and right by a pixel when the loading animation plays.

![image](https://github.com/ful1e5/Bibata_Cursor/assets/68742864/e9cf0573-f54e-4ae3-855d-94f62661eed2)
[Video demonstration](https://github.com/ful1e5/Bibata_Cursor/assets/68742864/ed60048a-bb07-4cbe-b0fc-7bd1904dcad0)

**Comment by ful1e5** [comment]
@Lassebq Thanks for letting me know. I'll fix it in the upcoming release.

**Comment by stanio** [comment]
I find some discrepancies in the `<svg width="..." height="..." viewBox="...'` attributes introduced with 4cb22d21bb596697e334047163665be604f627a9:

https://github.com/ful1e5/Bibata_Cursor/blob/04ca866c31b4510e73235108006698267359516b/svg/original/wait/wait-01.svg#L1

https://github.com/ful1e5/Bibata_Cursor/blob/04ca866c31b4510e73235108006698267359516b/svg/original/wait/wait-02.svg#L1

https://github.com/ful1e5/Bibata_Cursor/blob/04ca866c31b4510e73235108006698267359516b/svg/original/wait/wait-53.svg#L1

https://github.com/ful1e5/Bibata_Cursor/blob/04ca866c31b4510e73235108006698267359516b/svg/original/wait/wait-54.svg#L1

These don't appear corrected in 62e20e8b2a19d42ee90f43dbd9de8d834a1e468d.

`original/left_ptr_watch` has similar problem:

https://github.com/ful1e5/Bibata_Cursor/blob/04ca866c31b4510e73235108006698267359516b/svg/original/left_ptr_watch/left_ptr_watch-01.svg#L1

https://github.com/ful1e5/Bibata_Cursor/blob/04ca866c31b4510e73235108006698267359516b/svg/original/left_ptr_watch/left_ptr_watch-02.svg#L1

**Comment by RejZoRSheep** [comment]
What is happening with this bug? It's really annoying and baffling how someone managed to break animation of ALL cursors and it now somehow can't be fixed in a month time. It's just bunch of cursors?!

**Comment by stanio** [comment]
> `original/left_ptr_watch` has similar problem

It also shows some deformation to the spinning wheel.  The following makes it round:

```diff
--- a/svg/original/left_ptr_watch/left_ptr_watch-01.svg
+++ b/svg/original/left_ptr_watch/left_ptr_watch-01.svg
@@ -1,8 +1,10 @@
 <svg width="256" height="256" viewBox="0 0 256 256" fill="none" xmlns="http://www.w3.org/2000/svg">
 <path d="M72.7113 37.7047L58.4991 24.8108L58.5 44.0004L58.5055 164L58.5061 176.678L70.2338 171.863L115.08 153.451L115.282 153.368L115.48 153.274L159.629 132.431L171.135 126.999L161.711 118.449L72.7113 37.7047Z" fill="#00FF00" stroke="#0000FF" stroke-width="16"/>
+<g transform="scale(1 0.9635886)">
 <path d="M118 242.125C157.27 242.125 188.5 209.028 188.5 168.961C188.5 128.894 157.27 95.7969 118 95.7969C78.7304 95.7969 47.5 128.894 47.5 168.961C47.5 209.028 78.7304 242.125 118 242.125Z" fill="#FF0000" stroke="#0000FF" stroke-width="16"/>
 <path d="M123.081 118.539L118 168.961L166.345 174.261C167.012 167.639 166.422 160.945 164.609 154.561C162.796 148.178 159.795 142.229 155.778 137.054C151.761 131.88 146.805 127.582 141.195 124.405C135.585 121.228 129.43 119.235 123.081 118.539Z" fill="#F05024" fill-opacity="0.8"/>
 <path d="M114.609 219.537L118 168.961L166.493 172.498C166.047 179.139 164.352 185.625 161.504 191.583C158.655 197.542 154.71 202.857 149.892 207.225C145.074 211.593 139.478 214.928 133.425 217.041C127.371 219.153 120.977 220.002 114.609 219.537Z" fill="#FCB813" fill-opacity="0.8"/>
 <path d="M116.304 219.63L118 168.961L69.4185 167.192C69.1957 173.846 70.2317 180.48 72.4674 186.716C74.703 192.953 78.0945 198.669 82.4481 203.538C86.8018 208.408 92.0324 212.335 97.8413 215.096C103.65 217.857 109.924 219.398 116.304 219.63Z" fill="#7EBA41" fill-opacity="0.8"/>
 <path d="M118 118.261V168.961H69.3889C69.3889 162.303 70.6463 155.71 73.0892 149.559C75.5321 143.408 79.1128 137.819 83.6268 133.111C88.1407 128.403 93.4996 124.668 99.3973 122.12C105.295 119.573 111.616 118.261 118 118.261Z" fill="#32A0DA" fill-opacity="0.8"/>
+</g>
 </svg>
```

**Comment by ful1e5** [comment]
@RejZoRSheep 

> it now somehow can't be fixed in a month time. It's just bunch of cursors?

Apologies for the cursor animation issue. Dealing with personal challenges and managing multiple open-source projects without a full-time job has slowed me down. I appreciate your patience, and I'm actively working to fix the bug. Thanks for understanding.

**Comment by devlocalhost** [comment]
any update?

**Comment by pleskyjp** [comment]
I have downloaded v2.0.6 Bibata-Modern-Classic-Windows.zip and loading animation still shakes side to side by pixel

**Comment by RejZoRSheep** [comment]
How is this not fixed 4 months later? I get it that people maintain it for free, but come on?!

**Comment by stanio** [comment]
It appears fixed in the sources of the `main` branch: 3fe972d15fc333d8e23e2c5c4b4589cf39aefe33.  You should be able to try it on www.bibata.live as noted in the project README.

Alternatively, try building it yourself from sources.  This is what I'm doing for [my personal use](https://github.com/stanio/Bibata_Cursor/releases/latest), though I'm not using the original build infrastructure/scripts.

**Comment by MrtnKk** [comment]
Still love the Bibata-Modern-Classic cursors. Is there any chance we see another 2.0.7 release which finally fixes the shaking 'wait' mouse cursor?

**Comment by xfzv** [comment]
Fixed with [2.0.7](https://github.com/ful1e5/Bibata_Cursor/releases/tag/v2.0.7) on my end (`Bibata-Modern-Ice.tar.xz` / Linux)

**Comment by pleskyjp** [comment]
Fixed with 2.0.7 Bibata-Modern-Classic-Windows. Thank you 😍
